data/base-hunting.yaml

```diff
@@ -1354,3 +1354,175 @@ hunting_zones:
   - 11907
   - 13026
   - 13027
+
+  
+#################################################################
+# Added as of 3AUG2017 2335 EST; to be organized on next commit #
+#################################################################
+
+  ##Muspar'i ##the mobs not listed need some TLC from a mapdb guru
+  #
+  # 300-450
+  gam_chaga:
+  - 209
+  - 227
+  - 226
+  - 223
+  - 224
+  - 208
+  #
+  # 500-700 # needs a quick remap, nothing crazy.
+  ashu_shinvi:
+  - 166
+  - 162
+  #
+  # 80-100
+  sand_spider:
+  - 259
+  - 260
+  - 261
+  - 262
+  - 263
+  - 264
+  - 265
+  - 266
+  #
+  # 
+  #mottled_westanuryn 180-290
+  
+  #Leth Deriel
+  #DS leth 1200-250
+  #
+  # 175-220
+  twisted_dryad:
+  - 9766
+  - 9772
+  - 9773
+  - 9767
+  - 9774
+  - 9772
+  #
+  # 175-220-- epedia has the same values for these two, but they are in 
+  # two separate hunting areas, so this is likely not correct. These are the
+  # more difficult of the two, as well. Please test and update.
+  dryad_priestess:
```

Comment:
yes it is, but it breaks it up into two groups. 

Here is the rationalization: I don't want to break anybody before informing them that there is a change. There are two tiers here. E-pedia has them listed as having the same range, even though that's not the case. That's why I made a note of it. Yes.